### ISSUE TITLE
Issues with coreAnimation rendering

### ISSUE THREAD

**Issue by JuhaO81** [airbnb/lottie-ios#2030]
Some of our animations fails to render correctly with codeAnimation rendering engine. Issues are fixed when using mainThread. Animation can be requested from me by email. See my profile for email info.

**Comment by calda** [comment]
Can you please share the animations here, or email them to cal.stephens@airbnb.com

**Comment by miroslavkovac** [comment]
We are facing the same issue, sent an email with animation to cal.stephens@airbnb.com. Thanks for the help.

**Comment by calda** [comment]
Thanks @JuhaO81. I took a look at this issue and it appears to be a fundamental issue with the design of how `ShapeLayer`s were implemented in the Core Animation rendering engine (which is pretty different from the Main Thread rendering engine). I spent some time investigating and couldn't find a good, simple solution for this. For future reference, he's a simplified animation json that demonstrates the issue:

[2030.json.zip](https://github.com/airbnb/lottie-ios/files/11315183/2030.json.zip)
